README.md

```diff
@@ -39,7 +39,11 @@ module.exports = {
 };
 ```
 
-This extracts SourceMaps from all js files (including node_modules). This is not very performant, so you may want to only apply the loader to relevant files.
+`source-map-loader` extracts existing source maps from all JavaScript entries. This includes both inline source maps as well as those linked via URL. All source map data is passed to webpack for processing as per a chosen [source map style](https://webpack.js.org/configuration/devtool/) specified by the `devtool` option in [webpack.conf.js](https://webpack.js.org/configuration/).
```

Comment:
Haha, all my typos were with the **most** important terms (e.g. getting the proper casing on _webpack_ and getting the config filename name right). :-D